sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -7,7 +7,8 @@
             <v-btn
               color="primary"
               class="mr-5"
-              v-show="!isFirstPage && !readOnly"
+              v-if="!isReadOnly"
+              v-show="!isFirstPage"
```

Comment:
I believe that there is one AC missing. If the _submittingApplication_ is true, the draft button should not be visible, as per below AC
```
Do not display "save draft" while Application Submission is in Progress
```

Reply:
Flag added to hide "Save Draft" while submission in progress